### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Mickaellah**
Hi team,

Team Dong has just finished the home page without the top categories and the offers component. Please have a look at it and do not hesitate to comment or to suggest some changes.

Branch: https://github.com/onja-org/vamalaka-dong/tree/home-page
Clickup: https://app.clickup.com/t/45e562

Thanks in advance

### CODE REVIEW

**Review by Ny-Rianala — APPROVED** [review]

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -36,6 +28,7 @@ const ButtonStyled = styled.button<ButtonProps>`
     cursor: not-allowed;
     background-color: gray;
   }
+  box-shadow: -3px 3px rgba(21, 140, 177, 0.3), 3px -3px rgba(252, 70, 43, 0.3);
```
Comment (andrzej-onja) [inline src/components/Button/Button.tsx:31]: Nice simplification

File: src/components/DiscoveBanner/DiscoverBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            margin-block-start: 63px;
+        }
+    }
+
+    @media (min-width: 900px) and (max-width: 1200px) {
+        max-width: 950px;
+        margin-inline: auto;
+    }
+
+    @media (max-width: 900px) {
```
Comment (andrzej-onja) [inline src/components/DiscoveBanner/DiscoverBanner.tsx]: Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

File: src/components/DiscoveBanner/DiscoverBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            font-family: "Garamond";
+            font-size: 30px;
+            line-heigth: 33.75px;
+            color: #979797;
+            margin: 0;
+            margin-block-start: 63px;
+        }
+    }
+
+    @media (min-width: 900px) and (max-width: 1200px) {
```
Comment (andrzej-onja) [inline src/components/DiscoveBanner/DiscoverBanner.tsx]: Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -81,6 +81,14 @@ const DetailsStyles = styled.details`
       transform: rotate(180deg);
     }
   }
+
+  @media (max-width: 900px) {
```
Comment (andrzej-onja) [inline src/components/Footer/Footer.tsx]: Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

File: src/components/DiscoveBanner/DiscoverBanner.tsx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        font-size: 24px;
+        line-height: 27px;
+    }
+
+    button:last-child {
+        margin-inline-start: 45px;
+    }
+
+
+    @media (max-width: 900px) {
```
Comment (andrzej-onja) [inline src/components/DiscoveBanner/DiscoverBanner.tsx:90]: Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

File: src/components/Header/index.tsx [hunk 6]
```diff
@@ -43,6 +44,7 @@ const HeaderStyles = styled.header`
     gap: 50px;
 
     @media (max-width: 900px) {
```
Comment (andrzej-onja) [inline src/components/Header/index.tsx]: Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

File: src/components/LearnMore/LearnMore.tsx [hunk 7]
```diff
@@ -78,6 +77,12 @@ const LearnMoreContainer = styled.article`
             font-size: 25px;
         }
     }
+
+    @media (max-width: 900px) {
```
Comment (andrzej-onja) [inline src/components/LearnMore/LearnMore.tsx]: Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

File: src/components/PageFooter/PageFooter.tsx [hunk 8]
```diff
@@ -49,7 +49,7 @@ const FooterContainer = styled('div')`
   @media (min-width: 1114px) {
```
Comment (andrzej-onja) [inline src/components/PageFooter/PageFooter.tsx]: Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0

**Review by Tanosimboangy — APPROVED** [review]

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**